WindowsServerDocs/storage/storage-migration-service/known-issues.md

```diff
@@ -228,6 +228,28 @@ at Microsoft.StorageMigration.Proxy.Service.Transfer.TransferRequestHandler.Proc
 
 This was a code defect that would manifest if your migration account does not have at least Read permissions to the SMB shares. This issue was first fixed in cumulative update [4520062](https://support.microsoft.com/help/4520062/windows-10-update-kb4520062).
 
+Another possible cause might be insufficient access rights to the source file server. While examining the "Microsoft.StorageMigration.Proxy.Service.exe" process with Process Monitor you might see the below result:
+```
+Date: 6/04/2022 15:36:09,1943419
+Thread: 1688
+Class: File System
+Operation: CreateFile
+Result: PRIVILEGE_NOT_HELD
+Path: \\srv1.contoso.com\F$\\public
+Duration: 0.0002573
+
+Desired Access: Read Attributes, Read Control, Synchronize, Access System Security
+Disposition: Open
+Options: Synchronous IO Non-Alert, Open For Backup
+Attributes: N
+ShareMode: Read, Write
+AllocationSize: n/a
+Impersonating: CONTOSO\ServiceAccount
+OpenResult: PRIVILEGE_NOT_HELD
+```
+The actual operation being performed needs the "Open For Backup"-privileges on the source file server. Verify that your service account is granted the necessary permissions via the following Local Security Policy on this server or using a Group Policy Object:
```

Comment:
This is a great addition to the doc!

I recommend saying "source migration account", not "service account". A reader might think we're referring to the SMS orchestrator or proxy services themselves, which run as Network Service and shouldn't be touched.

Reply:
Issue solved in additional commit #[3cbfd76](https://github.com/thomasgeens/windowsserverdocs/commit/3cbfd76d1b440c1bbb2509e9a58c7d53e114e29b)